### PR TITLE
changed repo abbreviation to full name repository

### PR DESCRIPTION
**Pull request by gargprince1702**

### CODE REVIEW

**Comment by rahul799** [comment]
good job @gargprince1702.

File: README.md [hunk 1]
```diff
@@ -61,19 +61,19 @@ git push
 
 ### Create a branch to do your work.
 
-A good practice is to call the branch in the form of GH-<issue-number> followed by the title of the issue. This makes it easier to find out the issue you are trying to solve and helps us to understand what is done in the branch. Calling a branch my-work is confusing. Names of branch can not have a space, and should be replaced with a hyphen.
+A good practice is to call the branch in the form of GH- followed by the title of the issue. This makes it easier to find out the issue you are trying to solve and helps us to understand what is done in the branch. Calling a branch my-work is confusing. Names of a branch can not have space and should be replaced with a hyphen.
```
Comment (rahul799) [inline README.md:64]:
```suggestion
A good practice is to call the branch in the form of GH-<issue-number> followed by the title of the issue. This makes it easier to find out the issue you are trying to solve and helps us to understand what is done in the branch. Calling a branch my-work is confusing. Names of a branch can not have space and should be replaced with a hyphen.
```
I think it's fine.

Comment (gargprince1702) [reply README.md:64]: Yup @rahul799

Comment (rahul799) [reply README.md:64]: Prince please commit the changes requested.

Comment (gargprince1702) [reply README.md]: ok @rahul799

**Pull request merged**